src/main/scala/shine/DPIA/Types/DataType.scala

```diff
@@ -46,12 +46,6 @@ final case class DepArrayType private (size: Nat, elemFType: NatToData)
   override def toString: String = s"$size.$elemFType"
 }
 
-object DepArrayType {
-  def apply(size: Nat, f: DPIA.NatIdentifier => DataType): DepArrayType = {
```

Comment:
Was this unused, or why is this removed?

Reply:
There is now the following syntax instead:
```
size`.d`f
size`.d`{ x => f(x) }
```